assets/app/scripts/services/metrics.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MetricsService", function($http, $q, APIDiscovery, Logger) {
+    var COUNTER_TEMPLATE = "/counters/{containerName}%2F{podUID}%2F{metric}/data";
+    var GAUGE_TEMPLATE = "/gauges/{containerName}%2F{podUID}%2F{metric}/data";
+    var RATE_TEMPLATE = "/counters/{containerName}%2F{podUID}%2F{metric}/rate";
+
+    // URL template to show for each type of metric.
+    var templateByMetric = {
+      "cpu/usage": COUNTER_TEMPLATE,
+      "memory/usage": GAUGE_TEMPLATE,
+      "memory/page_faults": RATE_TEMPLATE,
```

Comment:
we aren't showing page faults though right, should we clean up what we arent using or do you think we would have these soon